Formula/databend.rb

```diff
@@ -0,0 +1,28 @@
+class Databend < Formula
+  desc "Elastic, reliable serverless data warehouse, offers blazing fast query"
+  homepage "https://databend.rs/"
+  url "https://github.com/datafuselabs/databend/archive/refs/tags/v0.6.36-nightly.tar.gz"
```

Comment:
Is there a stable version? We don't really package nightlies, along with the spirit of: https://docs.brew.sh/Acceptable-Formulae#stable-versions

Reply:
no they are all nightly versions.

Reply:
Then I don't think we can package this, sorry.